cli/UploadSoFiles.ts

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+import { Command, Option } from 'commander';
+import FormData from 'form-data';
+import fs from 'fs';
+
+interface UploadSoFilesOptions {
+  arch: 'x86' | 'x86_64' | 'arm64-v8a' | 'armeabi-v7a';
+  file: string;
+  token: string;
+  name: string;
+  api_key: string;
+}
+
+export const UploadSoFilesCommand = new Command();
+
+UploadSoFilesCommand.name('upload-so-files')
+  .addOption(
+    new Option('-arch, --arch <value>', 'arch')
+      .choices(['x86', 'x86_64', 'arm64-v8a', 'armeabi-v7a'])
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option(
+      '-f, --file <path>',
+      'The path of the symbol files in Zip format',
+    ).makeOptionMandatory(),
+  )
+  .addOption(new Option('--api_key <value>', 'Your App key').makeOptionMandatory())
+  .addOption(
+    new Option('-t, --token <value>', 'Your App Token')
+      .env('INSTABUG_APP_TOKEN')
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option('-n, --name <value>', 'The app version name')
+      .env('INSTABUG_APP_VERSION_NAME')
+      .makeOptionMandatory(),
+  )
+  .action(function (this: Command) {
+    const options = this.opts<UploadSoFilesOptions>();
+    UploadSoFiles(options);
+  })
+  .showHelpAfterError();
+
+const UploadSoFiles = async (opts: UploadSoFilesOptions) => {
+  const fileName = opts.file;
```

Comment:
Just a thought: If we are unsure that the file might not be there, how about we check if it exists and log an error in this case. Otherwise let's leave it as is.